pynicotine/slskmessages.py

```diff
@@ -1864,7 +1864,7 @@ def makeNetworkMessage(self):
                self.packObject(NetworkIntType(self.token)) +
                self.packObject(NetworkIntType(len(filelist))))
         for i in filelist:
-            msg += (chr(1) +
+            msg += (bytes(1) +
```

Comment:
The argument to bytes(), if it is an integer, is interpreted as the length of the bytes object, not its contents (https://docs.python.org/3/library/functions.html#func-bytes -> https://docs.python.org/3/library/stdtypes.html#bytearray) This does not create a bytes object of 1 byte with value 1, but a byte object of length 1 with value 0. Similarly, bytes(0) creates an empty bytes. The correct way to do this is bytes([1]) or bytes((1,)).

Reply:
BTW, ignore this, this has already been fixed in the `python3` branch.